docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/application_page/node--cgov-application-page--ncids-default-no-left-nav--full.html.twig

```diff
@@ -0,0 +1,6 @@
+<!-- ********************************* BEGIN Page Content ********************************** -->
+<article>
+  <div class="main-content type" id="main-content">
+    {{ content.field_application_module }}
+  </div>
+</article>
```

Comment:
This stuff needs to not be here for NCIDS. (well, yes, `{{ content.field_application_module }}` should be here.)

Reply:
This is where maybe you want the node--cgov-applicaiton-page--full to be just `{{ content.field_application_module }}` and update page--cgov-application-page--OLDSTUFF to include this markup.